internal/docs/exported_fields.go

```diff
@@ -5,19 +5,27 @@
 package docs
 
 import (
+	"errors"
 	"fmt"
 	"sort"
 	"strings"
 
+	"github.com/Masterminds/semver/v3"
+
 	"github.com/elastic/elastic-package/internal/fields"
+	"github.com/elastic/elastic-package/internal/packages"
 )
 
+var semver3_2_3 = semver.MustParse("3.2.3")
```

Comment:
3.2.3 has been already released (and 3.3.0), we would need to start doing this at least on 3.3.1.

Reply:
I've set the requirement to 3.3.1. It doesn't require a change to the spec, as both **value** and **example** are already allowed keys in the fields.yml file